### PR TITLE
[REF] pylint.cfg: Enable new checks from pylint-odoo-1.2.0

### PR DESCRIPTION
**Pull request by moylop260**
- Add new checks from new version of [pylint-odoo-1.2.0](https://github.com/oca/pylint-odoo) just in modules changed, following the [roadmap](https://github.com/OCA/pylint-odoo/issues/14)

### CODE REVIEW

**Comment by moylop260** [comment]
New checks from pylint-odoo enabled in MQT
@max3903 @dreispt @pedrobaeza

**Comment by dreispt** [comment]
It is unpleasant to suddenly turn branches red, and people will complain.
I feel that there should be an opt in period, decided per Project/PSC, so that a clean up can be done, and pending PRs rebuilt.

**Comment by moylop260** [comment]
@dreispt 
This PR won't show builds red because are enabled just for modules changed in PR and showed as warning's (don't affect the travis build result)

**Comment by moylop260** [comment]
Wait me a second... you are right!
I need added them to [travis_run_pylint_beta.cfg](https://github.com/OCA/maintainer-quality-tools/blob/master/travis/cfg/travis_run_pylint_beta.cfg) to avoid affect the travis build result

**Comment by moylop260** [comment]
@dreispt 
Thanks for feedback
Now the `.travis.yml` is without changes because don't affect the pylint errors counter and you can see the final message of the build with
```
Next checks are still in beta they won't affect your build status for now:
api-one-deprecated, api-one-multi-together, attribute-deprecated,
class-camelcase, copy-wo-api-one, create-user-wo-reset-password, 
dangerous-filter-wo-user, deprecated-module, duplicate-id-csv, 
duplicate-xml-fields, duplicate-xml-record-id, incoherent-interpreter-exec-perm,
javascript-lint, manifest-author-string, manifest-deprecated-key,
manifest-required-author, manifest-required-key, method-required-super,
missing-newline-extrafiles, missing-readme, no-utf8-coding-comment,
openerp-exception-warning, redundant-modulename-xml, rst-syntax-error,
too-complex, translation-field, translation-required, use-vim-comment,
wrong-tabs-instead-of-spaces, xml-syntax-error
```

**Comment by pedrobaeza** [comment]
Why the coverage drop?

**Comment by moylop260** [comment]
@pedrobaeza 
It's was a fails of coverage I press the button of rebuild in travis. Now the [last build](https://coveralls.io/builds/6199450) have the result: `COVERAGE REMAINED THE SAME AT 89.184%`

**Comment by dreispt** [comment]
LGTM :+1:

**Comment by nhomar** [comment]
:+1:

File: travis/cfg/travis_run_pylint_pr.cfg [hunk 1]
```diff
@@ -18,47 +18,71 @@ manifest_deprecated_keys=description,active
...
 #   openerp-exception-warning - R8101
+#   redundant-modulename-xml - W7909
 #   rst-syntax-error - E7901
 #   too-complex - C0901
 #   translation-field - W8103
+#   translation-required - C8107
 #   use-vim-comment - W8202
+#   wrong-tabs-instead-of-spaces
 #   xml-syntax-error - E7902
-#   eval-used - W0123
```
Comment (pedrobaeza) [inline travis/cfg/travis_run_pylint_pr.cfg]: Why do you remove this check?

Comment (moylop260) [inline travis/cfg/travis_run_pylint_pr.cfg]: It was changed by sort alphabetical to `e*` section

File: travis/cfg/travis_run_pylint_pr.cfg [hunk 2]
```diff
@@ -18,47 +18,71 @@ manifest_deprecated_keys=description,active
...
+    attribute-deprecated,
     class-camelcase,
+    create-user-wo-reset-password,
     copy-wo-api-one,
     dangerous-filter-wo-user,
     deprecated-module,
+    duplicate-id-csv,
+    duplicate-xml-fields,
     duplicate-xml-record-id,
+    eval-used,
```
Comment (moylop260) [inline travis/cfg/travis_run_pylint_pr.cfg]: @pedrobaeza 
It's here `eval-used`

**Comment by pedrobaeza** [comment]
:+1:

**Comment by moylop260** [comment]
Thanks @nhomar @pedrobaeza @dreispt

**Pull request merged**